src/cephadm/cephadm

```diff
@@ -2951,6 +2959,11 @@ def command_bootstrap():
         out = cli(['config', 'get', 'mgr', 'mgr/dashboard/ssl_server_port'])
         port = int(out)
 
+        # Open dashboard port
+        fw = Firewalld()
+        fw.open_ports([port])
+        fw.apply_rules()
```

Comment:
In the bootstrap case, you also need to open 

1. ~~mgr/prometheus exporter (9283)~~
2. dashboard's HTTP port (8080)

Reply:
In the bootstrap case the port used by default(8080) or explicitly set for the dashboard is obtained and opened.

take a look a few lines before:

```
        logger.info('Fetching dashboard port number...')
        out = cli(['config', 'get', 'mgr', 'mgr/dashboard/ssl_server_port'])
        port = int(out)
```

Reply:
right, but we have `mgr/dashboard/server_port` as well. unfortunately.

Reply:
https://github.com/ceph/ceph/pull/35746#discussion_r447601178